Airports/EGLL/Fixes.txt

```diff
@@ -14,6 +14,8 @@ LL01S N051.10.21.990 W000.41.08.430
 LLS01 N051.10.21.099 W000.41.08.086
 LLW02 N051.24.03.091 W000.43.36.089
 LLW03 N051.18.32.083 W000.45.56.094
+MAXIT N051.14.59.000 W000.33.43.000
+MODMI N051.14.01.000 W000.29.10.000
```

Comment:
Since we added MAXIT and MODMI into the airways, they're defined in Navdata/Fixes_UK.txt - so don't need to be defined here